ui/pages/send/send-content/add-recipient/domain-input.component.js

```diff
@@ -102,17 +106,18 @@ export default class DomainInput extends Component {
             'ens-input__wrapper--valid': hasSelectedAddress,
           })}
         >
-          <i
-            className={classnames('ens-input__wrapper__status-icon', 'fa', {
-              'fa-check-circle': hasSelectedAddress,
-              'fa-search': !hasSelectedAddress,
-            })}
-            style={{
-              color: hasSelectedAddress
-                ? 'var(--color-success-default)'
-                : 'var(--color-icon-muted)',
-            }}
-          />
+          {hasSelectedAddress ? (
+            <i
+              className="ens-input__wrapper__status-icon fa fa-check-circle"
+              style={{ color: 'var(--color-success-default)' }}
+            />
```

Comment:
I understand this PR is for fa-search only and fa fa-check-circle has been used in other files as well. But can we replace it here? Or should we file a separate PR for all fa-check-circle icons and include it there only? 

```suggestion
            <Icon
              name={ICON_NAMES.CONFIRMATION}
              color={ICON_COLORS.SUCCESS_DEFAULT}
              className="ens-input__wrapper__status-icon"
            />
```